Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml

```diff
@@ -0,0 +1,1930 @@
+          type: number
+      description: Get a quick indicator of the risk associated with a CVE.
+    - name: url
+      arguments:
+        - name: url
+          required: true
+          default: true
+          isArray: true
+          description: URL to get the reputation of
+      outputs:
+        - contextPath: DBotScore.Indicator
+          description: The indicator that was tested
+          type: string
+        - contextPath: DBotScore.Type
+          description: Indicator type
+          type: string
+        - contextPath: DBotScore.Vendor
+          description: Vendor used to calculate the score
+          type: string
+        - contextPath: DBotScore.Score
+          description: The actual score
+          type: number
+        - contextPath: URL.Malicious.Vendor
+          description: For malicious URLs, the vendor that made the decision
+          type: string
+        - contextPath: URL.Malicious.Description
+          description: For malicious URLs, the reason that the vendor made the decision
+          type: string
+        - contextPath: URL.Data
+          description: URL name
+          type: string
+        - contextPath: RecordedFuture.URL.riskScore
+          description: Recorded Future URL Risk Score
+          type: number
+        - contextPath: RecordedFuture.URL.riskLevel
+          description: Recorded Future URL Risk Level
+          type: string
+        - contextPath: RecordedFuture.URL.Evidence.rule
+          description: Recorded Risk Rule Name
+          type: string
+        - contextPath: RecordedFuture.URL.Evidence.mitigation
+          description: Recorded Risk Rule Mitigation
+          type: string
+        - contextPath: RecordedFuture.URL.Evidence.description
+          description: Recorded Risk Rule description
+          type: string
+        - contextPath: RecordedFuture.URL.Evidence.timestamp
+          description: Recorded Risk Rule timestamp
+          type: date
+        - contextPath: RecordedFuture.URL.Evidence.level
+          description: Recorded Risk Rule Level
+          type: number
+        - contextPath: RecordedFuture.URL.Evidence.ruleid
+          description: Recorded Risk Rule ID
+          type: string
+        - contextPath: RecordedFuture.URL.name
+          description: URL
+          type: string
+        - contextPath: RecordedFuture.URL.maxRules
+          description: Maximum count of Recorded Future URL Risk Rules
+          type: number
+        - contextPath: RecordedFuture.URL.ruleCount
+          description: Number of triggered Recorded Future URL Risk Rules
+          type: number
+      description: Get a quick indicator of the risk associated with a URL.
+    - name: recordedfuture-threat-assessment
+      arguments:
+        - name: context
+          auto: PREDEFINED
+          predefined:
+            - c2
+            - malware
+            - phishing
+          description: Context to use for verdict
```

Comment:
The only reason is to make it easier for customers, but I'd leave it up to our Product Manager (Mike) to decide any changes here. I personally don't see a problem with merging them (I'm guessing that it is possible to have predetermined input to a playbook, just like an action? Otherwise I need to add some check into the code to stop people from entering non-existing contexts).